### PR TITLE
support for using the mouse wheel to manipulate the gizmo

### PR DESCRIPTION
**Pull request by Yuki-Codes**
adds support for manipulating the gizmo with the mouse wheel while an axis is hovered.
also supports holding `Ctrl for small changes and `Shift` for large ones.

### CODE REVIEW

File: Brio/Core/ImGuizmoExtensions.cs
Comment (Minmoose) [inline Brio/Core/ImGuizmoExtensions.cs:1]: `Left Shift` and `Left Ctrl` are already in use.

Maby use `Right Shift` and `Right Ctrl` with `ImGui.IsKeyPressed()` instead?

**Review by Minmoose — CHANGES_REQUESTED** [review]
I like this idea, although the contol of it is a bit clunky,

I also encountered a bug where if you click and hold one of the gizmo's axes and then move the mouse wheel it soft-locks IMGUI.

**Comment by Yuki-Codes** [comment]
good calls, I've updated to use a configurable right shift/ctrl instead, and altered it so the scroll-wheel wont do anything if the mouse button is down.

**Review by Minmoose — APPROVED** [review]
Good deal, Thanks for the changes!

**Pull request merged**